### PR TITLE
WIP: bears: Add prerequirements installation instructions

### PR DESCRIPTION
**Pull request by Adrianzatreanu**
This PR is giving the user a helpful link in case the executable is failing to exist.

Currently WIP as it is only for one bear for test purposes, I'll continue working on others too
in different commits if it is okay. Looking forward to feedback.

Fixes https://github.com/coala-analyzer/coala-bears/issues/432

### CODE REVIEW

File: bears/dart/DartLintBear.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 
 @linter(executable='dartanalyzer',
+        executable_check_fail_info='Please see the Installation section of page'
+                                   'https://github.com/dart-lang/linter.',
```
Comment (sils1297) [inline bears/dart/DartLintBear.py]: can't you link directly to that section?

Comment (Adrianzatreanu) [inline bears/dart/DartLintBear.py]: eh i guess you're right, im going to change this as soon as my circleCI build finishes as i wanna see if it works.

**Comment by Adrianzatreanu** [comment]
@justuswilhelm tests were failing until now, i changed something to see how they're doing.

**Comment by Adrianzatreanu** [comment]
ah this fails because I havent used the coala-decorators module here, will fix this

**Comment by justuswilhelm** [comment]
@Adrianzatreanu You good on this?

**Comment by Adrianzatreanu** [comment]
yes, going to fix this for now:)

**Comment by Adrianzatreanu** [comment]
on second thought, do we really need this? Initially i thought it would help me on decentralizing bears, but it doesn't at all honestly. i'm still doing it if coala needs this, but i dont see when it would? all this does is add a link on executable missing (which we'd check without this attribute too) and copy pasting the first link on google about the executable ;p which probably any developer does anyway.
@sils1297 @Makman2 @justuswilhelm

**Comment by Adrianzatreanu** [comment]
closing for now, may continue later on.

**Pull request closed without merging**